scripts/spell_check/spelling.dat

```diff
@@ -2390,6 +2390,8 @@ dosent':doesn't
 dosen't:doesn't
 doub:doubt
 doulbe:double
+double click:double-click
+doubleclick:double-click
```

Comment:
I guess you don't want to add this one either as it's widely used in source code.
At least add  `:*` after it to avoid in-word detection (such as `mouseDoubleClickEvent` or `xxxx_cellDoubleClicked`)

again please run `./scripts/spell_check/check_spelling.sh` on the whole source code to see what happens.
thanks.